metrics/metrics.go

```diff
@@ -100,6 +119,8 @@ var (
 		DeschedulerStrategyDuration,
 		LoopDuration,
 		StrategyDuration,
+		LowNodeUtilizationThresholds,
+		LowNodeUtilizationClassification,
```

Comment:
This will not scale well for external plugins. Maybe a combination of init() and metrics registration within each plugin? Or, have you tried to invoke `RegisterMetrics` within the plugin instead?

Reply:
Done, I tried adding an optional `Metrics` interface to complement `Handle` without breaking backward compatibility